helpers_objectstorage.go

```diff
@@ -28,6 +30,11 @@ var bucketSchema = map[string]*schema.Schema{
 		Required: true,
 		Computed: false,
 	},
+	"access_type": {
+		Type:     schema.TypeString,
+		Computed: false,
```

Comment:
`Computed: false,` is the default. But it's included elsewhere, so at least it's consistent!